code/datums/outfits/jobs/civilian.dm

```diff
@@ -46,6 +46,7 @@
 	shoes = /obj/item/clothing/shoes/combat
 	glasses = /obj/item/clothing/glasses/sunglasses
 	gloves = /obj/item/clothing/gloves/thick/combat
+	l_hand = /obj/item/weapon/stamp/solgov
```

Comment:
if you changed the stamp you gotta change the type aswell to reflect that

Reply:
fixed